src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(oparation, number) {
+      oparation(number);
+
+      return this;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    add(number) {
+      calculator.result += number;
+    },
```

Comment:
right now we can execute this code:
![image](https://user-images.githubusercontent.com/37161315/172422339-eb604420-4a13-43fb-a08d-f2b1125f98d1.png)
i guess it should work a bit other way, we should change object that we called `operate` on, operation shouldn't be dependant on exact object, you can somehow pass needed object from operate to this actual `operation`